src/test/java/com/epam/tc/hw2/ex1/TestForFirstExercise.java

```diff
@@ -0,0 +1,112 @@
+        softly.assertThat(driver.getTitle()).isEqualTo("Home Page");
+
+        //Step 3. Perform login
+        WebElement findDropDown = driver.findElement(By.className("navbar-right"));
+        wait.until(ExpectedConditions.elementToBeClickable(findDropDown)).click();
+
+        WebElement username = driver.findElement(By.id("name"));
+        wait.until(ExpectedConditions.visibilityOf(username)).sendKeys("Roman");
+
+        WebElement password = driver.findElement(By.id("password"));
+        wait.until(ExpectedConditions.visibilityOf(password)).sendKeys("Jdi1234");
+
+        WebElement buttonEnter = driver.findElement(By.id("login-button"));
+        wait.until(ExpectedConditions.elementToBeClickable(buttonEnter)).click();
+
+        //4. Assert Username is loggined
+        WebElement logUsername = driver.findElement(By.id("user-name"));
+        wait.until(ExpectedConditions.visibilityOf(logUsername));
+        softly.assertThat(logUsername.getText()).isEqualTo("ROMAN IOVLEV");
+
+        //5. Assert that there are 4 items on the header section are displayed and they have proper text
+        List<WebElement> findHeaderItems = wait.until(ExpectedConditions.numberOfElementsToBeMoreThan(
+                By.cssSelector(".m-l8 > li"), 3));
+
+        List<String> expectedHeaderItems = newArrayList(
+                "HOME",
+                "CONTACT FORM",
+                "SERVICE",
+                "METALS & COLORS"
+        );
+
+        softly.assertThat(findHeaderItems)
+                .extracting(WebElement::getText)
+                .containsExactlyElementsOf(expectedHeaderItems);
+
+        //6. Assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> imageItems = wait.until(
+                ExpectedConditions.numberOfElementsToBeMoreThan(
+                By.cssSelector("div[class*=benefit] > span[class*=icons-benefit]"), 3));
+
+        softly.assertThat(imageItems).hasSize(4).allMatch(WebElement::isDisplayed);
+
+        //7. Assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<WebElement> textUnderImage = wait.until(
+                ExpectedConditions.numberOfElementsToBeMoreThan(By.className("benefit-txt"), 3));
+
+        List<String> expectedTextUnderImages = newArrayList(
+                "To include good practices\nand ideas from successful\nEPAM project",
+                "To be flexible and\ncustomizable",
+                "To be multiplatform",
+                "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…"
+        );
+
+        softly.assertThat(textUnderImage)
+                .extracting(WebElement::getText)
+                .containsExactlyElementsOf(expectedTextUnderImages);
+
+        //8. Assert that there is the iframe with “Frame Button” exist
+        softly.assertThat(driver.findElement(By.id("frame")).isDisplayed()).isTrue();
```

Comment:
If element does not exist then you'll have NoSuchElementException and test fail right in place
Instead you can use `.findElements(...)` and check size of resulting list is not 0

\+ same for the next assertion